common/metrics/otel_metrics_handler.go

```diff
@@ -96,7 +96,7 @@ func (omp *otelMetricsHandler) Timer(timer string) TimerMetric {
 	}
 
 	return TimerMetricFunc(func(i time.Duration, t ...Tag) {
-		c.Record(context.Background(), i.Nanoseconds(), tagsToAttributes(omp.tags, t, omp.excludeTags)...)
+		c.Record(context.Background(), i.Milliseconds(), tagsToAttributes(omp.tags, t, omp.excludeTags)...)
```

Comment:
Would this make the unit issue go away for otel? As it will show up as second as well like tally?

Reply:
Well I can change to `Seconds()`, then I guess we don't have to change the unit in our dashboard with switching to otel. 

But I don't quite like that since the code will be very confusing, and the logic should not depend on how dashboard is setup.

Reply:
Synced offline. The value here must be an int. The precision will be too low if we using Seconds() here.